plugins/cliconf/iosxr.py

```diff
@@ -341,6 +366,21 @@ def commit(self, comment=None, label=None, replace=None):
             cmd_obj["answer"] = "y"
         self.send_command(**cmd_obj)
 
+    def commit_confirmed(self):
+        cmd = "commit confirmed"
+
+        if self.get_option("commit_confirmed_timeout"):
+            cmd += " {0}".format(self.get_option("commit_confirmed_timeout"))
+        if self.get_option("commit_confirmed_label"):
+            cmd += " label {0}".format(
+                self.get_option("commit_confirmed_label")
+            )
+        if self.get_option("commit_confirmed_comment"):
+            cmd += " comment {0}".format(
+                self.get_option("commit_confirmed_comment")
+            )
```

Comment:
if commit confirmed is defined we fire - `commit confirmed`
if commit_confirmed_timeout is defined we fire - `commit confirmed <timeout>`
if commit_confirmed_label & timeout is defined we fire - `commit confirmed <timeout> label <lbl>`
if commit_confirmed_comment & label & timeout is defined we fire - `commit confirmed <timeout> label <lbl> comment <cmnt>`
All the options are can be used independently
which makes it support 7.0.2 / 7.2.2 / 7.4.4
And, yes the supported version would be 7.4.x and every option together is supported in that version
all the variables are independent and the user is supposed to decide which combination to use to produce the right command

Reply:
Since we know this feature will not work below some version, should we gate the functionality, or update the docs to list the lowest version we support?

Reply:
We should update the docs as per the lowest version supported